duckdb-vortex/src/include/vortex_common.hpp

```diff
@@ -0,0 +1,65 @@
+
+#include "vortex.h"
+
+using duckdb::make_uniq;
```

Comment:
As a general guideline, it is discouraged to use `using` statements in headers as those spill into all source files including the header.